datasets/br/br_slavery/crawler.py

```diff
@@ -0,0 +1,48 @@
+import csv
+from typing import Dict
+
+from rigour.mime.types import CSV
+
+from zavod import Context, helpers as h
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    entity = context.make("Company")
+    entity.id = context.make_slug(row.pop("ID"))
```

Comment:
I'm not sure I trust their IDs to remain consistent year on year with people being added/removed, meaning an ID applying to one person now could apply to a different person in future, which is really bad.

See below for my suggestion to use their tax number in the id. And if tax number is blank, you could use name and address in `make_id` although I doubt they'd be blank in this list.